### ISSUE TITLE
feat: eslint-plugin-no-autofix support 3rd party plugin

### ISSUE THREAD

**Issue by barroudjo** [aladdin-add/eslint-plugin#61]
Would it be possible to use your plugin with non core rules (for instance, a react specific rule: react-hooks/exhaustive-deps) ?

Thanks for your work !

**Comment by aladdin-add** [comment]
atm it does not, I was thinking of some hack ways to support it. :)

**Comment by aladdin-add** [comment]
I use a proxy to support it. please try v1.0.0 ❤️

**Comment by barroudjo** [comment]
Wow that was fast ! I'll try it tomorrow.

**Comment by barroudjo** [comment]
Sorry, it doesn't work at least in some cases...
Here's the error it gives me:  
1:1  error  Definition for rule 'no-autofix/react-hooks/exhaustive-deps' was not found  no-autofix/react-hooks/exhaustive-deps

Here is my .eslintrc in case it can help !
```json
{
  "env": {
    "browser": true
  },
  "parser": "@typescript-eslint/parser",
  "parserOptions": {
    "project": "./tsconfig.json"
  },
  "plugins": ["no-autofix", "@typescript-eslint", "react-hooks"],
  "rules": {
    "no-useless-escape": 0,
    "space-before-function-paren": ["error", "always"],
    "react/prop-types": 0,
    "semi": [2, "always"],
    "no-extra-semi": 2,
    "@typescript-eslint/strict-boolean-expressions": 0,
    "@typescript-eslint/no-explicit-any": "warn",
    "@typescript-eslint/no-floating-promises": 0,
    "@typescript-eslint/promise-function-async": 0,
    "@typescript-eslint/restrict-template-expressions": ["error", { "allowNumber": true }],
    "react-hooks/rules-of-hooks": "error",
    "react-hooks/exhaustive-deps": "warn",
    "no-autofix/react-hooks/exhaustive-deps": "warn"
  },
  "extends": ["standard-react", "standard-with-typescript"],
  "settings": {
    "import/resolver": {
      "node": {
        "extensions": [".js", ".jsx", ".ts", ".tsx"]
      }
    },
    "react": {
      "version": "detect"
    }
  }
}

```

 And I'm using the latest versions of eslint, here are my versions:
```json
{
    "@typescript-eslint/eslint-plugin": "^2.19.2",
    "@typescript-eslint/parser": "^2.19.2",
    "eslint": "^6.8.0",
    "eslint-config-standard-react": "^9.2.0",
    "eslint-config-standard-with-typescript": "^13.0.0",
    "eslint-plugin-import": "^2.20.1",
    "eslint-plugin-no-autofix": "^1.0.0",
    "eslint-plugin-node": "^11.0.0",
    "eslint-plugin-promise": "^4.2.1",
    "eslint-plugin-react": "^7.18.3",
    "eslint-plugin-react-hooks": "^2.3.0",
    "eslint-plugin-standard": "^4.0.1"
  }
```

**Comment by aladdin-add** [comment]
thanks for letting me know, I reverted that change, and published v1.0.1.

I didn't noticed a proxy object is not allowed being exported in eslint plugin. so it seems there is no easy way to fix it. 😥

**Comment by aladdin-add** [comment]
released in 1.1.0 https://www.npmjs.com/package/eslint-plugin-no-autofix/v/1.1.0
